projects/tests/test_create_delete_environments.py

```diff
@@ -0,0 +1,132 @@
+from django.contrib.auth import get_user_model
+from django.contrib.messages import get_messages
+from django.test import TestCase
+
+from apps.models import Apps, JupyterInstance, Subdomain
+from projects.models import Environment, Project
+from projects.views import can_model_instance_be_deleted
+
+User = get_user_model()
+
+test_user = {"username": "foo1", "email": "foo@test.com", "password": "bar"}
+test_superuser = {"username": "superuser", "email": "superuser@test.com", "password": "bar"}
```

Comment:
This is just a style thing, because they are global constants
```suggestion
TEST_USER = {"username": "foo1", "email": "foo@test.com", "password": "bar"}
TEST_SUPERUSER = {"username": "superuser", "email": "superuser@test.com", "password": "bar"}
```